src/sentry/integrations/opsgenie/__init__.py

```diff
@@ -1 +1,7 @@
+# mypy: ignore-errors
+from sentry.rules import rules
+
+from .actions import OpsgenieNotifyTeamAction
 from .integration import *  # noqa: F401,F403
+
+rules.add(OpsgenieNotifyTeamAction)
```

Comment:
I'd just `type: ignore` line 7 rather than the whole file. We generally want to preserve type checking wherever we can.

Reply:
You can also add an `__all__` declaration to remove the `#noqa` comment and be explicit about what we want to export from this file. :+1:

Reply:
For some reason, type : ignore on line 7 wasn't working yesterday... but I'll try it again!